apps/timetable/admin/ui/js/index.js

```diff
@@ -77,14 +77,14 @@ define(['gh.core', 'gh.constants', 'gh.listview', 'gh.admin-listview', 'gh.admin
      */
     var renderLoginForm = function() {
         $('#gh-subheader, #gh-content-description').height(350);
-        gh.utils.renderTemplate($('#gh-login-template'), {
+        gh.utils.renderTemplate('admin-login-form', {
             'gh': gh
-        }, $('#gh-subheader'));
-
-        // Bind the validator to the login form
-        $('.gh-signin-form').validator({
-            'disable': false
-        }).on('submit', doLogin);
+        }, $('#gh-subheader'), function() {
+            // Bind the validator to the login form
+            $('.gh-signin-form').validator({
```

Comment:
We might want to use `.onAvailable()` here as well before binding listeners to the form.